src/menu_nanos.c

```diff
@@ -0,0 +1,54 @@
+#include <os_io_seproxyhal.h>
+#if defined(TARGET_NANOS)
+
+#include "menu.h"
+#include "getVersion.h"
+#include "glyphs.h"
+
+// Here we define the main menu, using the Ledger-provided menu API. This menu
```

Comment:
We could probably remove some of these comments. They are from original Sia app and I guess no longer needed. I would keep just the comments on the struct